src/app-layout/__tests__/split-panel-provider.test.tsx

```diff
@@ -96,19 +96,14 @@ describe.each(['bottom', 'side'] as const)('position=%s', position => {
       </SplitPanelProvider>
     );
 
-    expect(reportSize).toHaveBeenCalledTimes(2);
+    expect(reportSize).toHaveBeenCalledTimes(position === 'bottom' ? 2 : 1);
```

Comment:
The refactored optimized side position rendering, but not bottom, hence the difference in behavior.

Might do more on fixing the bottom position another time